### DISCUSSION TITLE
payments flow to support incremental authorization

### DISCUSSION THREAD

**Discussion by euluizaraujo** [juspay/hyperswitch#3026] [Q&A]
Hello how are you? I would like to better understand how this new incremental payment functionality works and its applications. Could you help me with this question? @sai-harsha-vardhan 

https://github.com/juspay/hyperswitch/pull/3009

**Comment by sai-harsha-vardhan** [comment]
Hey @euluizaraujo 
Incremental authorizations allows you to increase authorized amount for a payment before capturing it
- This is possible only for card payments and specific networks, with some criteria specific to connector
- Incremental Authorizations can be done multiple times and maximum attempts is again based on connector 
- Total amount capturable would be sum of initial authorized amount and all the incremented additional amounts
- Incremental authorization is not possible if merchant and cardholder are in country with Strong Customer Authentication (SCA) requirements

The feature is not yet completely developed; we will be releasing docs once it's done. Thanks for your interest!

**Comment by euluizaraujo** [comment]
What would be the use cases for this functionality? I think this way the understanding would be clearer.

**Reply by sai-harsha-vardhan** [reply]
One of the usecase would be;
**Traveling**
When booking a hotel room, you can use incremental authorization if a traveler initially reserves a room for a certain number of nights but decides to extend their stay. In this case, an incremental authorization can be requested to cover the additional nights' charges.

**Comment by euluizaraujo** [comment]
@sai-harsha-vardhan  today I have the following case: SaaS platform charges the customer monthly for the number of API requests, but this number is only closed at the end of the month, i.e., the customer registers a card with a small verification value (US$1) and the platform charges at the end of the month according to usage.

Would this scenario already be possible with the hyperswitch?

**Reply by euluizaraujo** [reply]
@sai-harsha-vardhan

**Reply by sai-harsha-vardhan** [reply]
Hey @euluizaraujo, This use case falls under `Mandates`, which allows you to save the customer's card with us and then charge monthly based on the billing cycle you have. You can find more details about Recurring Payments here -> https://hyperswitch.io/docs/features/mandates
Thank you, Hope this helps!
